frontend/src/services/fetch/fetchers/applicationFetcher.ts

```diff
@@ -1,21 +1,53 @@
-import { APIResponse } from "src/types/apiResponseTypes";
+import {
+  ApplicationDetailApiResponse,
+  ApplicationFormDetailApiResponse,
+  ApplicationResponseDetail,
+  ApplicationStartApiResponse,
+} from "src/types/applicationResponseTypes";
 
 import { fetchApplicationWithMethod } from "./fetchers";
 
-interface ApplicationStartResponse extends APIResponse {
-  data: {
-    application_id: string;
-  };
-}
-
-// make call from server to API to start an application
 export const handleStartApplication = async (
   competitionID: string,
-): Promise<ApplicationStartResponse> => {
+): Promise<ApplicationStartApiResponse> => {
   const response = await fetchApplicationWithMethod("POST")({
     subPath: `start`,
     body: { competition_id: competitionID },
   });
 
-  return (await response.json()) as ApplicationStartResponse;
+  return (await response.json()) as ApplicationStartApiResponse;
+};
+
+export const getApplicationDetails = async (
```

Comment:
the usage here sort of demonstrates the limitations of this layer of abstraction. At this point we'd probably be good just using the `fetchApplicationWithMethod` function directly within the components. Slighty harder to test but not much

Reply:
I actually like how it provides the consistent names for the data and endpoint that is getting retrieved, even if it is extra verbose. You can see that from the endpoints as well, so don't have a strong opinion here.

Reply:
yeah, something to discuss later on, we can see how things progress